.github/workflows/e2e-test-linux-vm.yaml

```diff
@@ -5,47 +5,73 @@ on:
     branches:
       - master
   pull_request:
+      branches:
+      - master
   workflow_dispatch:
 
 env:
   TARGET_ORG: actions-runner-controller
-  CLUSTER_NAME: e2e-test
-  RUNNER_VERSION: 2.302.1
-  IMAGE_REPO: "test/test-image"
+  IMAGE_NAME: "arc-test-image"
+  IMAGE_VERSION: "test"
+  TEST_CONFIG_URL: "https://github.com/actions-runner-controller/arc_e2e_test_dummy"
```

Comment:
I think there's value in making those workflow_dispatch inputs so that we can override their values without creating a PR

Reply:
updated.